galaxy/datatype_rdata_xcms/rdata_xcms_datatype.py

```diff
@@ -14,30 +14,74 @@
 log = logging.getLogger(__name__)
     
 
-class RdataXcmsRaw( Binary ):
+class RdataXcmsGeneric (RData):
+    """Class describing a sequence"""
```

Comment:
This should ideally be in Galaxy core. Maybe we can create a galaxy_datatypes conda package that we encourage people to install to get all the sniffer working by default?